crates/sui-indexer-alt-e2e-tests/tests/transactional_tests.rs

```diff
@@ -161,6 +161,7 @@ async fn cluster(config: &OffChainConfig) -> Arc<OffchainCluster> {
                     pruner: Some(pruner),
                     ..Default::default()
                 }),
+                tx_balance_changes: Some(ConcurrentLayer::default()),
```

Comment:
This should already be enabled as part of `IndexerConfig::for_test` (all pipelines should be enabled for testing) -- is that not the case?